examples/external/java/read-keys-example/lib/README.md

```diff
@@ -0,0 +1 @@
+Copy your libeelektra4j-VERSION.jar file to this directory. The same jar name should be specified in the pom.xml file [here](../pom.xml).
```

Comment:
Please move this file one directory-level up. Otherwise people directly browsing through the folders might not see it.